.storybook/themes/commerce.scss

```diff
@@ -3,43 +3,589 @@
+		// alignment.  It accounts for a minimum width that has (--global-spacing-5 / 2) margins
+		//
+		"layout-content-scale-width":
+			min(
+				calc(100vw - var(--global-spacing-5)),
+				calc(var(--layout-content-max-width) / var(--layout-max-width) * 100%)
+			),
+		"primary-color": var(--global-neutral-10),
+		"primary-font-family": "Inter",
+		"secondary-font-color": "Lora",
+	),
 	$tokens: (
 		default: (
-			components: (
-				stack: (
-					gap: 1rem,
-				),
-				stack-divider-vertical: (
-					border: none,
-					border-bottom-width: 2px,
-					border-bottom-style: solid,
-					border-bottom-color: #712d2d,
-					height: 1px,
+			// DESIGN ASSUMPTION: Attribution does not require component styling.
+			components:
+				(
+					body: (
+						background-color: var(--global-color-white),
+						color: var(--global-color-black),
+						font-family: var(--primary-font-family),
+						font-size: var(--global-font-size-4),
+						line-height: var(--global-line-height-4),
+					),
+					button: (
+						border: 1px solid transparent,
+						border-radius: var(--global-border-radius-1),
+						font-size: var(--global-font-size-3),
+						line-height: var(--global-line-height-3),
+						font-weight: var(--global-font-weight-7),
+						gap: var(--global-spacing-2),
+					),
+					button-small: (
+						padding: var(--global-spacing-1) var(--global-spacing-3),
+					),
+					button-medium: (
+						padding: var(--global-spacing-3),
+					),
+					button-large: (
+						padding: var(--global-spacing-3),
+						font-size: var(--global-font-size-4),
+						line-height: var(--global-line-height-4),
+					),
+					// DESIGN QUESTION:
+					// Please confirm this style in Storybook for a link button.
+					button-link:
+						(
+							background-color: transparent,
+							color: var(--global-neutral-10),
+							font-weight: var(--global-font-weight-4),
+							text-decoration: underline,
+						),
+					button-primary: (
+						background-color: var(--global-neutral-10),
+						color: var(--global-white),
+					),
+					button-primary-hover: (
+						background-color: var(--global-neutral-7),
+					),
+					button-primary-reverse: (
+						background-color: transparent,
+						border-color: var(--global-neutral-10),
+						color: var(--global-neutral-10),
+					),
+					button-primary-reverse-hover: (
+						border-color: var(--global-neutral-7),
+						color: var(--global-neutral-7),
+					),
+					button-secondary: (
+						background-color: var(--global-white),
+						color: var(--global-neutral-10),
+					),
+					button-secondary-hover: (
+						background-color: var(--global-neutral-7),
+						color: var(--global-white),
+					),
+					button-secondary-reverse: (
+						border-color: var(--global-white),
+						background-color: transparent,
+						color: var(--global-white),
+					),
+					button-secondary-reverse-hover: (
+						border-color: var(--global-neutral-7),
+						background-color: transparent,
```

Comment:
No need for `background-color` here as it will inherit the same value from the non hover state

Reply:
removed